libafl_frida/src/asan/asan_rt.rs

```diff
@@ -593,7 +593,7 @@ impl AsanRuntime {
 
                     let _ = [<$lib_ident:snake:upper _ $name:snake:upper _PTR>].set(unsafe {std::mem::transmute::<*const c_void, extern "C" fn($($param: $param_type),*) -> $return_type>(target_function.0)}).unwrap_or_else(|e| println!("{:?}", e));
 
-                    #[expect(non_snake_case)]
+                    #[allow(non_snake_case)]
```

Comment:
Why not expect?

Reply:
Because its a macro and not *all* use cases are non_snake_case.